### PR TITLE
Cookierelease: setup src folder structure

### PR DESCRIPTION
**Pull request by bobleesj**

### CODE REVIEW

File: src/diffpy/nmf_mapping/__init__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# https://github.com/diffpy/diffpy.nmf_mapping/graphs/contributors
+#
+# See LICENSE.rst for license information.
+#
+##############################################################################
+
+"""Run NMF analysis on PDF and XRD data"""
+
+# package version
+from diffpy.nmf_mapping.version import __version__
```
Comment (bobleesj) [inline src/diffpy/nmf_mapping/__init__.py:19]: nmf_mapping used.

File: src/diffpy/nmf_mapping/nmf_mapping_code.py [hunk 2]
```diff
@@ -11,11 +11,12 @@
 import numpy as np
 import pandas as pd
 from bg_mpl_stylesheets.styles import all_styles
-from diffpy.utils.parsers.loaddata import loadData
 from scipy import interpolate
 from sklearn.decomposition import NMF, PCA
 from sklearn.exceptions import ConvergenceWarning
 
+from diffpy.utils.parsers.loaddata import loadData
```
Comment (bobleesj) [inline src/diffpy/nmf_mapping/nmf_mapping_code.py:18]: done by pre-commit

File: src/diffpy/nmf_mapping/version.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+"""Definition of __version__."""
+
+#  We do not use the other three variables, but can be added back if needed.
+#  __all__ = ["__date__", "__git_commit__", "__timestamp__", "__version__"]
+
+# obtain version information
+from importlib.metadata import version
+
+__version__ = version("diffpy.nmf_mapping")
```
Comment (bobleesj) [inline src/diffpy/nmf_mapping/version.py:24]: nmf_mapping

**Review by bobleesj — COMMENTED** [review]
Not ready for a PR at the moment.

**Review by sbillinge — COMMENTED** [review]
this looks good.  It is failing precommit for what may be a typo in run.py but I am not sure.  Tests are now running.  Since this is mostly moving files, I am ok merging this and then debugging on the next PR, but it is showing as a draft PR indicating that you don't want me to merge it yet.  Just let me know.

**Comment by bobleesj** [comment]
@sbillinge Yes, Please merge. I will look into the details (`run.py`, etc) tomorrow.

**Pull request merged**